Source/Charts/Data/Implementations/Standard/CombinedChartData.swift

```diff
@@ -28,6 +28,11 @@ open class CombinedChartData: BarLineScatterCandleBubbleChartData
     {
         super.init(dataSets: dataSets)
     }
+
+    public required init(arrayLiteral elements: ChartDataSetProtocol...)
```

Comment:
have you checked if any tricky bug or trap (like index to a wrong position) regards CombinedChartData collection conformance? CombinedChartData is a wrapper putting other data together like barData, lineData.

Reply:
I made sure that all the new Swift methods/properties have the same implementations as the existing ones, so this should be an issue.

Reply:
sure. It will take more time, need to catch up the dataset protocol first.